src/NzbDrone.Common/Http/HttpRequest.cs

```diff
@@ -54,6 +54,7 @@ public Uri Url
         public Dictionary<string, string> Cookies { get; private set; }
         public bool StoreResponseCookie { get; set; }
         public TimeSpan RateLimit { get; set; }
+        public HttpRequestProxySettings Proxy {get; set;}
```

Comment:
yes, but atm it's injected to HttpRequest via a HttpRequestInjector, which was never the intended usage.